src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="big form"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class name 'big form' is not descriptive of the content. Consider using a class name that describes the purpose or function of the input, such as 'search-input-large'.